src/FilterData.js

```diff
@@ -13,31 +16,53 @@ class FilterData {
   }
 
   getUser(userID) {
-    return Promise.all([this.user, this.getAllTripsForUser(userID)])
+    return Promise.all([this.user, this.allTrips, this.allDestinations])
       .then(promises => {
-        return new User(promises[0].id, promises[0].name, promises[0].travelerType, promises[1])
+        console.log(promises[0])
+        return new User(promises[0].id, promises[0].name, promises[0].travelerType, this.getAllTrips(promises[0].id, promises[1], promises[2], promises[0]))
       })
   }
 
-
-  getAllTripsForUser(userID) {
```

Comment:
I see that this method is deleted in this PR and is commented out below.  Are you still marinating on how to put this into place?